raiden/tests/scenarios/bf6_stress_hub_node.yaml

```diff
@@ -52,7 +52,7 @@ nodes:
     8:
       matrix-server: {{ matrix_servers[3] }}
     9:
-      matrix-server: {{ matrix_servers[4] }}
+      matrix-server: {{ matrix_servers[3] }}
```

Comment:
I'm not sure I understood why this change is relevant for the tests

Reply:
It's not. It's actually reverting an older commit when we skipped `transport05` because it was not working properly. Now the scenario uses all transport servers again like it used to do.